### ISSUE TITLE
Add a loop/replay button

### ISSUE THREAD

**Issue by theridon** [TeamNewPipe/NewPipe#8647]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

This app has just about everything else, it's great. But for some reason there is no replay/loop button/functionality? Can we just get a simple button below the video that would let it replay videos, such as for listening to music in a car? Thank you.

### Why do you want this feature?

So the app is better

**Comment by SameenAhnaf** [comment]
Rejected. See: https://github.com/TeamNewPipe/NewPipe/issues/6914

However, you can still use this functionality from notification player. Go to `Settings`>`Appearence`>`Player Notification`>`4th/ 5th Action Button`> Select `Repeat`

**Comment by theridon** [comment]
I disable notifications from my apps, so I can't use that feature. Plus it's quite inconvenient to have to drag down the notification bar for such a basic, integral feature. Can we just get a button that's usable when normally using the app?

**Comment by opusforlife2** [comment]
> I disable notifications from my apps

In version 0.20.0, Newpipe's player was rewritten and made wholly dependent on a unified player service, which needs a foreground notification to anchor it in memory.

I don't think we support use cases where notifications for the app are disabled. But I'll let a developer weigh in on this.

**Comment by mhmdanas** [comment]
I wholeheartedly agree that it doesn't make sense for the looping functionality to be present only in the media notification, not the app itself. It's quite inconvenient. Like I stated in https://github.com/TeamNewPipe/NewPipe/issues/6914#issuecomment-899027896, adding a loop button in the player would be a good idea in my opinion.

**Comment by Stypox** [comment]
> I don't think we support use cases where notifications for the app are disabled. But I'll let a developer weigh in on this.

Foreground apps work just fine if you disable their notifications ;-)

**Comment by opusforlife2** [comment]
I agree with Stypox's solution in https://github.com/TeamNewPipe/NewPipe/issues/6914#issuecomment-1190142193.

**Comment by merlin555** [comment]
I'm for it too!!

**Comment by Stypox** [comment]
I'm moving this out of the "Rewrite" project. The point of that project is to track the work that needs to be done to migrate the app and rewrite parts of it that are bad. We will indeed rethink about the whole player structure and placement of buttons throughout the app, and this issue is relevant on that end, but it falls within a bigger set of issues related to the UX of the player.
